frontend/src/Explorer/View/Transaction.purs

```diff
@@ -70,7 +70,9 @@ summaryItems :: CTxSummary -> Language -> SummaryItems
 summaryItems (ctxSum@CTxSummary txSummary) lang =
     [ { label: translate (I18nL.tx <<< I18nL.txTime) lang
       , value: let  dateFormat = translate (I18nL.common <<< I18nL.cDateFormat) lang
-                    dateValue = fromMaybe noData <<< prettyDate dateFormat $ txSummary ^. ctsTxTimeIssued
+                    mDate      = txSummary ^. ctsTxTimeIssued
```

Comment:
If something is changed it's going to affect readability, until you get used to it. 

> but if you think dateValue was not readable or understandable before, we can do this change

Yes, I was faced with compiler errors, started moving things around and then stopped, thought "why am I complicating things?" and wrote it like that. Compiler agreed.

Seems you like composition a lot? Is that it?